src/Dapr.Actors.AspNetCore/ActorsEndpointRouteBuilderExtensions.cs

```diff
@@ -198,6 +200,20 @@ private static IEndpointConventionBuilder MapActorHealthChecks(this IEndpointRou
         return new Tuple<string, byte[]>(serializedHeader, responseMsgBody);
     }
 
+    /// <summary>
+    /// Generate exception info
+    /// </summary>
+    /// <param name="ex">Exception of the method.</param>
+    /// <returns>Exception info string</returns>
+    private static string GetExceptionInfo(Exception ex) {
```

Comment:
You should encode this as a Base64 string and then return the `Array<byte>` value for the header.